### ISSUE TITLE
Jest button function tests

### ISSUE THREAD

**Issue by FireLemons** [rubyforgood/casa#2446]
**Description**
Write tests for `showBtn`, `hideBtn`, `disableBtn`, and `enableBtn` in `app/javascript/src/casa_case.js`  
  
see `app/javascript/__tests__/case_contact.test.js` for an example

**Comment by himanshu007-creator** [comment]
Hi! @FireLemons , i haven't done this before but would like to give it a try🙂

**Comment by FireLemons** [comment]
@himanshu007-creator I'm trying this myself and it's way harder than I thought it'd be. Let me know if you get stuck

**Comment by compwron** [comment]
@himanshu007-creator or @FireLemons are either of you working on this? Feel free to re-add yourself if you want- otherwise I am moving it back to To Do :)

**Comment by himanshu007-creator** [comment]
Yes I am trying to work on this issue. Please give me some time ( till Thursday ). If it won't be possible for me, I'll let you know

**Comment by himanshu007-creator** [comment]
@compwron 
i have created test for showBtn , but for others , there are some issues, please give a look on this one
<pre>
/**
 * @jest-environment jsdom
 */
const showbtn = require("./showbtn");

test("showbtn works properly", () => {
  const elem = document.createElement("div");
  const nw = ["d-none"];
  elem.classList.add("d-none");
  expect(showbtn(elem)).toEqual(expect.not.arrayContaining(nw));
});

</pre>

**Comment by compwron** [comment]
@himanshu007-creator this looks feasible. @FireLemons / @RomanTurner can you offer advice here?

**Comment by RomanTurner** [comment]
@FireLemons and I wrote something similar for the Notifier test case.

**Comment by FireLemons** [comment]
shush

**Comment by himanshu007-creator** [comment]
I am willing to work. I had some health issues.
Sorry for the inconvenience

**Comment by compwron** [comment]
Health is important! No worries, I'm just checking in.

**Comment by FireLemons** [comment]
oh wait theta's the unassigned message. I thought it was the unassigned warning

**Comment by FireLemons** [comment]
I was trying to reply to the bot to extend the time
